storybook/storybook-css/src/components/Heading/story-template.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2022 Gemeente Utrecht
+ * Copyright (c) 2022 Frameless B.V.
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import React from 'react'
+
+export const argTypes = {
+  textContent: {
+    table: {
+      type: { summary: 'string' },
+    },
+  },
+  level: {
+    options: [1, 2, 3, 4, 5, 6],
+    control: 'select',
+    table: {
+      type: { summary: 'number' },
+      defaultValue: { summary: '1' },
+    },
+  },
+  styleLevel: {
+    options: [1, 2, 3, 4, 5, 6],
+    control: 'select',
+    table: {
+      type: { summary: 'number' },
+      defaultValue: { summary: '1' },
+    },
+  },
+}
+
+export const defaultArgs = {
+  textContent: 'Jouw typograaf biedt mij zulke exquise schreven!',
+  level: 1,
+  styleLevel: 1,
+}
+
+export const heading2Args = {
+  textContent: 'Jouw typograaf biedt mij zulke exquise schreven!',
+  level: 2,
+  styleLevel: 2,
+}
```

Comment:
I think we can make the `headingXArgs` objects a bit more DRY:

```suggestion
export const heading2Args = {
  ...defaultArgs,
  level: 2,
  styleLevel: 2,
}
```